var/spack/repos/builtin/packages/gimp/package.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and otherargs
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Gimp(AutotoolsPackage):
+    """GIMP is a cross-platform image editor available for GNU/Linux,
+    macOS, Windows and more operating systems. It is free software,
+    you can change its source code and distribute your changes.
+
+    Whether you are a graphic designer, photographer, illustrator, or
+    scientist, GIMP provides you with sophisticated tools to get your job
+    done. You can further enhance your productivity with GIMP thanks to
+    many customization options and 3rd party plugins."""
+
+    homepage = "https://www.gimp.org"
+    url = "https://download.gimp.org/gimp/v2.10/gimp-2.10.32.tar.bz2"
```

Comment:
Probably need to add a `url_for_version` too, let me know if you need help

Reply:
Will do!